### ISSUE TITLE
Format:date with example renders time in addition to date

### ISSUE THREAD

**Issue by ilya-spv** [swagger-api/swagger-ui#5215]
### Q&A (please complete the following information)
 - OS: Windows
 - Browser: Chrome
 - Version: 72
 - Method of installation: visual studio code extension (openapi-designer)
 - Swagger-UI version: [e.g. 3.21.0]
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
I use a visual studio extension to render things on the fly (openapi-designer) and have checked that it uses the latest version of swagger-ui

Example Swagger/OpenAPI definition:
```yaml
startDate:
   type: string
   format: date
   example: 2018-12-25
```

### Describe the bug you're encountering
There was recently a [pull request](https://github.com/swagger-api/swagger-ui/pull/5024) to make automatic example if it is not present. That works fine, but when I have my own example, then the rendered date for some reason includes timestamp as well:
```
"startDate": "2018-12-25T00:00:00.000Z",
```

### To reproduce...

1. Render YAML with type: string and format: date
2. Open an example value in the browser

### Expected behavior
Expected to see only date in example
```
"startDate": "2018-12-25",
```

**Comment by shockey** [comment]
Indeed, the [specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#data-types) is pretty clear on this.

**Comment by robert-hebel-sb** [comment]
hey @parthpetkar 👋 
are you able to reproduce the issue?
I've tried it with the following example and bug seems to be fixed already

<img width="1520" height="628" alt="Image" src="https://github.com/user-attachments/assets/5adc112c-403a-4634-8fa9-064bdfae7f4c" />
